go.mod

```diff
@@ -132,3 +132,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/equinix/equinix-sdk-go => github.com/equinix/equinix-sdk-go v0.39.1-0.20240506214523-caea8fda690e
```

Comment:
Replace directives are good for validating unreleased SDK changes but shouldn't be merged.  Once the SDK changes have been validated they should be included in an official SDK release and then the version should be bumped in this repo/PR.

Reply:
Yes, correct. I was submitting the PR just to get eyes on it but did not intend it to be approvable until the go.mod/go.sum were corrected to point to a released version of the `equinix-sdk-go` package.